lms/djangoapps/courseware/views/views.py

```diff
@@ -283,8 +283,9 @@ def course_info(request, course_id):
             # redirect to the dashboard page.
             if isinstance(access_response, StartDateError):
                 start_date = strftime_localized(course.start, 'SHORT_DATE')
-                params = urllib.urlencode({'notlive': start_date})
```

Comment:
According to my understandings, with `QueryDict`, encoding will be used from `settings.DEFAULT_CHARSET` (`ascii`, `utf-8`, etc – default is `utf-8`), 
and in former approach we are explicitly telling to encode using `utf-8` as python uses `ascii` as default charset.

So, I am ok either way.